src/modules/Dropdown/Dropdown.js

```diff
@@ -950,7 +952,7 @@ export default class Dropdown extends Component {
     }
 
     if (!newSelectedIndex || newSelectedIndex < 0) {
-      newSelectedIndex = enabledIndicies[0]
```

Comment:
`newSelectedIndex` is undefined in the case above causing this to set with enabledIndicies[0].